packages/riverpod_lint/lib/src/assists/convert_to_stateless_base_widget.dart

```diff
@@ -147,89 +150,250 @@ class ConvertToStatelessBaseWidget extends RiverpodAssist {
           .whereType<MethodDeclaration>()
           .firstWhereOrNull((element) => element.name.lexeme == 'createState');
       if (createStateMethod != null) {
-        builder.addDeletion(createStateMethod.sourceRange);
+        builder.addDeletion(createStateMethod.sourceRange.getExpanded(1));
```

Comment:
Could you add a comment about what this is about? The line isn't very clear to me